cli/build/build.go

```diff
@@ -63,6 +63,10 @@ func Run(ctx *context.Ctx) error {
 		}
 
 		ctx.Build.Spec = filepath.Base(ctx.Build.Spec)
+
+		if common.ContainsUpperSymbols(ctx.Build.Spec) {
+			return fmt.Errorf("Name of the .rockspec file mustn't contain capital letters")
```

Comment:
How about to add this check for `cartridge pack` too? `--spec` flag is used for pack too.

Reply:
``cartridge pack`` calls this function (``build.Run``) so checking is already enabled:

```
~/repos/cartridge-cli   fix-build-capital-letters-project-name  ./cartridge pack tgz MyApp --spec=MyApp/MyApp-scm-1.rockspec 
   • Packing MyApp into tgz   
   • Temporary directory is set to /Users/aleksei.romanov/.cartridge/tmp/pack-zfumiuqa4g
   • Initialize application dir
   • Build application in /Users/aleksei.romanov/.cartridge/tmp/pack-zfumiuqa4g/package-files/MyApp
   ⨯ Name of the .rockspec file mustn't contain capital letters
```

Reply:
Ok, thanks